api/bazel/api_build_system.bzl

```diff
@@ -74,6 +74,23 @@ def api_go_grpc_library(name, proto, deps = []):
         ],
     )
 
+def api_cc_grpc_library(name, proto, deps = []):
+    cc_grpc_library(
+        name = name,
+        srcs = [proto],
+        deps = deps,
+        proto_only = False,
+        grpc_only = True,
+        visibility = ["//visibility:public"],
```

Comment:
Should we name this `_api_cc_grpc_library` (same with others in this file) to discourage use outside of the `.bzl`?

Reply:
Done in 2740d3eb6924e5aa3105b456f9438441fe3be6d4